compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -169,6 +169,11 @@ def TileAndDistributeToWorkgroups :
     Pass<"iree-codegen-tile-and-distribute-to-workgroups", "IREE::HAL::ExecutableVariantOp"> {
   let summary = "Tile and distribute operations to workgroups";
   let constructor = "mlir::iree_compiler::createTileAndDistributeToWorkgroupsPass()";
+  let options = [
+    Option<"maxWorkgroupParallelDims", "max-workgroup-parallel-dims", "int32_t",
```

Comment:
Please dont hard code to 3... use kNumMaxParallelDims.

Reply:
`kNumMaxParallelDims` is a constexpr constant. Can I reference that in TD files?

Reply:
If it is defined before the place the generated header file is included, then you can

Reply:
I dont know what happens if the default isnt set...

Reply:
I believe the Options are like `Optional` types, hence the `hasValue()` method. If I set this to empty, it uses the default value in the constructor `createTileAndDistributeToWorkgroupsPass()`. 

As stated earlier, I could not figure out how to get the `kNumMaxParallelDims` constant here. Its defined in Codgen/Utils.h, but the generated `compiler/src/iree/compiler/Codegen/Passes.h.inc` does not have any `#includes` in it and I cannot seem to add it. 

I couldn't find another example of using a constexpr in a TD file. Everything else uses literals. Any ideas?

Reply:
it should work but if you wanted to use `kNumMaxParallelDims` you just need to make sure it is part of a header included before `compiler/src/iree/compiler/Codegen/Passes.h.inc` (for instance you would need to make sure this gets included in `compiler/src/iree/compiler/Codegen/Passes.h`)

Reply:
Looked through how to not hard-coded... too complicated to manage the headers. This is just the default for command line option I guess. Please just hard-code it to 3 for now....